src-os/ptime_clock_stubs.c

```diff
@@ -181,9 +226,34 @@ CAMLprim value ocaml_ptime_clock_current_tz_offset_s (value unit)
        (dd == -1 || dd >  1 /* year wrap */) ? dm - (24 * 60) :
        dm /* same day */;
 
-  value some = caml_alloc (1, 0);
+  some = caml_alloc (1, 0);
   Store_field (some, 0, Val_int (dm * 60));
-  return some;
+  CAMLreturn(some);
```

Comment:
It's tidier this way, but just FTR this isn't strictly necessary, since no allocations (and hence Gc) can occur between the `caml_alloc` and the `return`.

Reply:
Indeed. Though unrelated to the initial aim @IndiscriminateCoding you can keep it as such, it will make the style uniform.